lib/beaker/options/presets.rb

```diff
@@ -110,7 +110,7 @@ def env_vars
       def presets
         h = Beaker::Options::OptionsHash.new
         h.merge({
-          :project                => 'Beaker',
+          :project                => ENV['JOB_NAME'] || 'Beaker',
```

Comment:
@anodelman JOB_NAME is set by Jenkins itself (it'd probably be helpful to make note of this in a comment)